README.md

```diff
@@ -1 +1,35 @@
-# MultiDID
+# multidid
+
+> A very compact representation of DIDs and DID URLs
+
+Multidid is a representation strategy for DIDs and DID URLs that is very compact and extensible. It allows any DID method to be represented as a byte string.
+
+## Format
+
+```
+<multicodec-did><method-multicodec><method-bytes><url-length><url-bytes>
+```
+
+Where:
+
+* `multicodec-did` - the value `0xd1d`
```

Comment:
@expede Should this be a varint? I see you are doing this for varsig, assume we want to do the same here? (Don't remember exactly why)

@bumblefudge unfortunately this would remove the niceness of reading `0xd1d` when seeing a binary DID 😅

Reply:
DID maxis everywhere, pour one out for the little 0xD1D that could've been... 🥤

Reply:
Wait a minute 🧐 

```
> u8a.toString(new Uint8Array([varint.decode(u8a.fromString('0d1d', 'base16'))]), 'base16')
'0d'
```

Looking at [The Table](https://github.com/multiformats/multicodec/blob/master/table.csv), looks like `0x0d` is not taken!

Reply:
> pour one out for the little 0xD1D that could've been

`0xd1d` is also hard to work with because it doesn't break on a byte boundary. `0x0d1d` would work, though!

A lot of the multicodec table has little puns like this that get scrambled during serialization. Especially since you then need to encode those bytes to something _else_ to transmit them as a string (e.g. ASCII, base58, base64) for JSON, you'll almost never see the hex.

> Should this be a varint?

I'm certainly not the multicodec table, but I believe that would make life easiest in the long term, yes. If someone later registers `0x0d1dab`, then you can distinguish it from `0x0d1d` in the same way that we can distinguish 123 from 1234.

But I believe that @rvagg is the right person to ask (sorry for all the pings this week! 🙏)

Reply:
Makes sense. Will update this.